src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/FlatFWTest.java

```diff
@@ -123,6 +213,13 @@ public void shouldFailToSetFixed2BeforeFixed1() throws Exception
                 .fixed2(10);
     }
 
+    @Test(expected = IndexOutOfBoundsException.class)
+    public void shouldFailToSetString1WhenDefaultingFixed2ExceedsMaxLimit()
```

Comment:
Remove this since already covered by shouldFailToSetFixed2WithInsufficientSpace()